### PR TITLE
Try Ninja on Windows

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -2,15 +2,14 @@ set PATH=%PREFIX%\cmake-bin\bin;%PATH%
...
 mkdir build && cd build
 
-cmake -G "%CMAKE_GENERATOR%" ^
+cmake -GNinja ^
 	  -D CMAKE_BUILD_TYPE=Release ^
 	  -D BUILD_SHARED_LIBS=ON ^
 	  -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
 	  -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
 	  -D ENABLE_ZLIB_COMPRESSION=ON ^
-	  -D CMAKE_BUILD_TYPE=Release ^
```
Comment (jakirkham) [inline recipe/bld.bat:11]: Do we still want this?

Comment (xhochy) [reply recipe/bld.bat:11]: This line was duplicated, it is still there.

Comment (jakirkham) [reply recipe/bld.bat:11]: Ah right 😅 Thanks for pointing that out 😄

**Comment by jakirkham** [comment]
Continuing in PR ( https://github.com/conda-forge/libssh2-feedstock/pull/35 ).

**Pull request closed without merging**